### PR TITLE
Updating Thai translation.

### PR DESCRIPTION
**Pull request by DatPanDat**
# SCP:SL Translation PR Template

## Language
Thai

## Type of change
- [ ] Added new language translation
- [x] Changed a currently existing translation

## Have you tested the changes you've made?
- [ ] Yes
- [x] No

# Checklist:
- [x] I have made sure to not commit changes to other stuff than my own or the default english one
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser)
- [x] I have made sure that I'm not submitting a duplicate translation 
- [x] I have not included any advertisements/pointers to non-game stuff other than author in the folder name

**New**
-WeaponManager.txt
-New SCP's and healing items

**Edited**
-Some old translation to make more senses.
-Some misspelling.

### CODE REVIEW

File: Translations/th/NewInventory.txt [hunk 1]
```diff
@@ -7,25 +7,29 @@
...
-9mm ที่ถูกทิ้งไว้:ไม่มีการอธิบาย.
-USP:ไม่มีการอธิบาย.
\ No newline at end of file
+ตัวปลดอาวุธ:คลิกเมาส์ซ้ายเพื่อจับ คลิกเมาส์ขวาเพื่อปล่อย \nกด E ค้างบนคนที่โดนจับแล้วอยู่ทีมเดียวกับคุณเพื่อปล่อยคนๆ นั้น
+7.62 ที่ถูกทิ้งไว้:ไม่มีการอธิบาย
+9mm ที่ถูกทิ้งไว้:ไม่มีการอธิบาย
+ยูเอสพี:ไม่มีการอธิบาย
+SCP-018:ซุปเปอร์บอลที่สามารถเด้งได้อย่างรุนแรง
+SCP-268:เมื่อสวมใส่ จะทำให้ล่องหนชั่วขณะ นอกจากว่าจะโดนขัดโดยการทำสิ่งต่าง ๆ ของผู้ใช้
+ยาแก้ปวด:ผ่อนคลายอาการบาดเจ็บเบื้องต้น
```
Comment (Dat-Joseph) [inline Translations/th/NewInventory.txt]: You need to switch line 34 with 35, take a look at the updated NewInventory.txt in the english translation.

Comment (DatPanDat) [reply Translations/th/NewInventory.txt]: Done.

**Comment by Dat-Joseph** [comment]
I just went over your PR, i noticed that your MainMenu and Abilities translation is outdated (missing and wrong lines). Check the current Eng. and correct it. We will then review your changes :)

**Comment by DatPanDat** [comment]
May I ask what is the "ARTIFICIAL HP"? Is it like overheal or something?

**Comment by Dat-Joseph** [comment]
Its additional HP. Lets say you have 100/100 HP, and a certain Item gives you 50HP as healing. It will then be added as artificial HP. Its depending on the item though.

**Comment by DatPanDat** [comment]
All done. Hopefully...

**Comment by Dat-Joseph** [comment]
Again i checked your Pull Request. Looks good! Dont forget to play Beta when it launches, cant tell when though. Check then if all works with your translation, if some things dont work you can just edit them and do a PR again :)

**Pull request merged**